.github/workflows/r.yml

```
@@ -86,6 +86,9 @@ jobs:
       - name: Dump install logs
         run: cat r/check/arrow.Rcheck/00install.out
         if: always()
+      - name: Dump test logs
+        run: cat r/check/arrow.Rcheck/tests/testthat.Rout
```

```suggestion
        run: cat r/check/arrow.Rcheck/tests/testthat.Rout*
```

_it turns out_ that if there's a test failure, the output is written to `r/check/arrow.Rcheck/tests/testthat.Rout.fail`

---

Ah, good catch.  Fixed.